### PR TITLE
Missing prequisite package 'openssl'

### PR DESCRIPTION
**Pull request by eldar101**
Even with 'openssl-devel' installed, I recieved an error while trying to run 'make e2e-tests". This was solved by installing 'openssl'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Flight Control is a service for declarative, GitOps-driven management of edge de
 ## Building
 
 Prerequisites:
-* `git`, `make`, and `go` (>= 1.20), `openssl-devel`, and `podman-compose`
+* `git`, `make`, and `go` (>= 1.20), `openssl`, `openssl-devel`, and `podman-compose`
```
Comment (hexfusion) [inline README.md:10]:
```suggestion
* `git`, `make`, and `go` (>= 1.21), `openssl`, `openssl-devel`, and `podman-compose`
```

**Review by hexfusion — COMMENTED** [review]
Thanks lets bump golang version as well since we are here then LGTM

**Review by hexfusion — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hexfusion) [reply README.md]: it is sort of interesting that devel doesn't require openssl :man_shrugging:

**Review by mangelajo — COMMENTED** [review]
Normally openssl-devel pulls openssl, but it is what it is.. :)

**Review by mangelajo — APPROVED** [review]

**Review by mangelajo — APPROVED** [review]

**Comment by hexfusion** [comment]
@eldar101 all commits must be signed then we can merge :pray:

**Pull request merged**